omod/src/main/webapp/pages/metadata/locations/location.gsp

```diff
@@ -9,7 +9,7 @@
 
     def parentLocationOptions = []
     existingLocations.each {
-        parentLocationOptions.push([ label: ui.escapeJs(ui.format(it)), value: it.id ])
+        parentLocationOptions.push([ label: ui.encodeJavaScript(ui.format(it)), value: it.id ])
```

Comment:
This was  change in my previous commit but i cant tell the problem with my git version control.  This change doesn't reflect in my local environment but it reflects on pushing though i first pull the latest changes.